iree/test/e2e/regression/BUILD

```diff
@@ -177,6 +177,7 @@ py_binary(
     generator_args = [
         "--lhs_rhs_type=%s" % lhs_rhs_type,
         "--shapes=large",
+        "--compilation_info=none",
```

Comment:
Wait, now that the new `NONE` enum value corresponds to the arg string value `""`,  which is the default value, don't you need to drop this flag here?  As it currently stands, iiuc, this `"none"` value is not one of the legal values so it should hit the `return ValueError` path that you just added?

Reply:
Oh hey this is exactly what the CI is complaining about :-P
```
generate_e2e_matmul_tests.py: error: argument --compilation_info: invalid choice: 'none' (choose from '', 'LLVMGPUMatmulSimt', 'LLVMGPUMatmulTensorCore')
```

Reply:
I missed this. Deleted the flag.